assets/js/lists/account-lists.tpl.liquid

```diff
@@ -46,7 +46,7 @@
             </span>
             <div  class="ajax-informer text-center" ng-class="{ 'fade': $ctrl.accountLists.productAdded }">{{ 'wishlist.general.added_to_cart' | t}}</div>
 
-            <table class="vc-product-table list-table">
+            <table class="vc-product-table list-table" aria-hidden="true">
```

Comment:
The more preferable way is to use role="presentation".

No issue will be raised on <table> used for layout purpose, i.e. when it contains a role attribute set to "presentation" or "none".